resources/META-INF/plugin.xml

```diff
@@ -536,6 +536,13 @@
     <annotator language="Dart" implementationClass="io.flutter.editor.FlutterEditorAnnotator"/>
     <errorHandler implementation="io.flutter.FlutterErrorReportSubmitter"/>
 
+    <toolWindow id="Flutter Preview" anchor="right" icon="FlutterIcons.Flutter_13"
+                factoryClass="io.flutter.view.preview.PreviewViewFactory"
+                conditionClass="io.flutter.view.FlutterViewCondition"/>
```

Comment:
Let's switch this to a new `FlutterPreviewCondition`, and have that also check the state of the preference to enable 'Flutter Preview'.

Reply:
Done.